lib/core/nav/HeaderNav.js

```diff
@@ -218,6 +218,12 @@ class HeaderNav extends React.Component {
       (env.translation.enabled
         ? this.props.language + '/versions.html'
         : 'versions.html');
+    let headerClass;
+    if (siteConfig.headerIcon) {
+      headerClass="headerTitleHidden"
```

Comment:
This is not standard JS style. Should use semicolons, single quotes and space around operators.

Reply:
Consider simplifying to:

```
const headerClass = siteConfig.headerIcon ? 'headerTitleHidden' : 'headerTitle';
```

Reply:
I figured out how to use Prettier in VSCode and am now running `yarn ci-check` to check this before pushing to Circle CI. 📈

Changed to ternary, good suggestion 👍